docs/Contributing/Releasing-Fleet.md

```diff
@@ -141,21 +135,17 @@ must be created and relevant changes cherry-picked onto that branch:
    Action](https://github.com/fleetdm/fleet/actions/workflows/goreleaser-snapshot-fleet.yaml) will
    be invoked to push a container image for QA with `fleetctl preview` (eg. `fleetctl preview --tag patch-fleet-v4.3.1`).
 
-4. Check in the GitHub UI that Actions ran successfully for this branch and perform [QA smoke
-   testing](https://github.com/fleetdm/fleet/blob/main/.github/ISSUE_TEMPLATE/smoke-tests.md).
+4. The DRI for creating the patch release branch cherry checks in the GitHub UI that Actions ran successfully for this branch.
+
+5. The DRI for creating the patch release branch notifies the [DRI for release testing/QA](https://fleetdm.com/handbook/product#rituals) that the branch is available for completing [smoke tests](https://github.com/fleetdm/fleet/blob/main/.github/ISSUE_TEMPLATE/smoke-tests.md).
+
+6. The DRI for release testing/QA makes sure the standard release instructions at the top of this document are followed. Be sure that modifications to the changelog and config files are commited _on the `patch-*` branch_.
 
-5. Follow the standard release instructions at the top of this document. Be sure that modifications
-   to the changelog and config files are commited _on the `patch-*` branch_. When the patch has been
-   released, return to finish the following steps.
+7. The DRI for release testing/QA notifies the [DRI for the release ritual](https://fleetdm.com/handbook/engineering#rituals) that the patch release is ready. The DRI for the release ritual releases the patch.
 
-6. Cherry-pick the commit containing the changelog updates into a new branch, and merge that commit
-   into `main` through a Pull Request.
+8. The DRI for creating patch the release branch cherry-picks the commit containing the changelog updates into a new branch, and merge that commit into `main` through a Pull Request.
 
-7. **Important!** Manually check the database migrations. Any migrations that are not cherry-picked in a
-   patch must have a _higher_ timestamp than migrations that were cherry-picked. If there
-   are new migrations that were not cherry-picked, verify that those migrations have higher
-   timestamps. If they do not, submit a new Pull Request to increase the timestamps and ensure that
-   migrations are run in the appropriate order.
+9. **Important!** The DRI for creating patch the release branch manually checks the database migrations. Any migrations that are not cherry-picked in a patch must have a _higher_ timestamp than migrations that were cherry-picked. If there are new migrations that were not cherry-picked, verify that those migrations have higher timestamps. If they do not, submit a new Pull Request to increase the timestamps and ensure that migrations are run in the appropriate order.
```

Comment:
...**the** patch ~~the~~...

Reply:
> must have a _higher_ timestamp

Seems like timestamps would be _earlier_ or _later_ vs. _lower_ or _higher_.

Reply:
✅ 

@lukeheath I updated "higher timestamps" to "later timestamps" because I assume higher timestamps come after lower timestamps. Is this correct?

Reply:
> higher timestamps come after lower timestamps

Correct!